app.js

```diff
@@ -65,3 +65,24 @@ const booksCollection = new BooksCollection();
 
 booksCollection.init();
 booksCollection.displayBooks();
+const listButton = document.querySelector('.booklist');
+const addNewButton = document.querySelector('.addition');
+const contactButton = document.querySelector('.contact');
+const awesomeBooksSection = document.querySelector('.show-books');
+const addNewSection = document.querySelector('.add-newbook');
```

Comment:
- [ ] Good job working on some of the project requirements but I noticed your design is missing a few things that need to be addressed before your work gets approval. If you carefully look at the template design or wireframe it displays the date and time in all page tabs but for your design, it is not displaying the date and time. This was supposed to be done using [luxon](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time)

This is how your work looks:



![image](https://user-images.githubusercontent.com/79058364/243112621-fb43bb3a-e731-4895-8109-dc6cfd6c0523.png)

Please follow this wireframe to display the date and time:

![DATE AND TIME ](https://github.com/kamranasif506/Awesome-books/assets/79506105/506a66fe-e959-412a-b33b-def74caa00d1)

Remember this should appear on all three pages of your tabs.


- [ ] Secondly it would be nice to properly align the content on your contact page to properly match the design template you can do this by adjusting the margin and padding.

your page content should be centered properly. Please follow this template design:

![Contact page ](https://github.com/kamranasif506/Awesome-books/assets/79506105/528e841b-9bf1-4aaf-8b04-bc5c8861714f)